lib/Controller/ArgumentValueResolver/WebsiteConfigValueResolver.php

```diff
@@ -18,22 +18,24 @@
 
 use Pimcore\Config;
 use Symfony\Component\HttpFoundation\Request;
-use Symfony\Component\HttpKernel\Controller\ArgumentValueResolverInterface;
+use Symfony\Component\HttpKernel\Controller\ValueResolverInterface;
 use Symfony\Component\HttpKernel\ControllerMetadata\ArgumentMetadata;
 
 /**
  * Adds support for type hinting controller actions and getting the current website config.
- *
  */
-final class WebsiteConfigValueResolver implements ArgumentValueResolverInterface
+final class WebsiteConfigValueResolver implements ValueResolverInterface
```

Comment:
this class not marked as internal so removing a method is considered a BC break.

Reply:
Ok. regression of https://github.com/pimcore/pimcore/pull/12911/files#diff-0a9c0cad482ae4e69e0fb9297c5d4e613892cd9cd09e93a68554a150c2c9abccL24. maybe we should mark it as internal again.

Reply:
@dvesh3 Mark it again https://github.com/pimcore/pimcore/pull/15796/commits/f7cad40f0541168c961d74ce61c43a9b02cfe9be